caffe2/CMakeLists.txt

```diff
@@ -1162,6 +1162,17 @@ if(USE_DISTRIBUTED)
     target_compile_definitions(torch_cpu PRIVATE
       USE_TENSORPIPE
     )
+    # This is needed for torch/csrc/distributed/rpc/macros.h to work
+    # in other than tensorpipe_agent source files, like rref_impl.cpp
```

Comment:
in other than tensorpipe_agent source files, like rref_impl.cpp 

-> in libtorch_distributed_sources source files, e.g., rref_impl.cpp. See tools/build_variables.bzl

Reply:
I'm sorry, is it a suggestion to edit? replace the last line?

Reply:
Yep, please feel free to rephrase. I just wanna help make this comments more accurate. Because not all "other than tensorpipe_agent source files" need this. IIUC, only files not in the list below need that.

https://github.com/pytorch/pytorch/blob/b80c6f863f2327c712c478f67c248b94d66b65ac/caffe2/CMakeLists.txt#L365-L372